launch-checklist.md

```diff
@@ -1 +1,20 @@
+# Replicache Launch Checklist
```

Comment:
Nice. Can you create a new `doc/` folder and put it there? I'm going to be putting all the doc there.